### ISSUE TITLE
[Todoist] Apple SSO not working

### ISSUE THREAD

**Issue by max-zinn** [raycast/extensions#14012]
### Extension

https://www.raycast.com/doist/todoist

### Raycast Version

1.80.0

### macOS Version

14.1 (23B2073)

#### Additional info

I wanted to try a different todo extension for raycast and installed todoist for that. Unfortunately I wasn't able to log in. My next steps included downloading the mac app of todoist as well, but that didn't work either (Because of that I assume an issue on todoist and created a ticket [833496](https://get.todoist.help/hc/de/requests/833496)).
I decided to also create a ticket through the extension bug-report as well to give you a quick heads-up and in case they changed their api and you need to take actions here.

#### Error:
```
{"_isRayError":true,"code":16,"title":"Could not initialize OAuth","name":"Error","message":"OAuth request creation is not available when command is launched in background","stack":"\n    at Zn (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/@raycast/server/index.js:5:6290)\n    at j9\n    at _r (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/@raycast/api/index.js:8:2593)\n    at ray-navigation-stack\n    at Eo (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/@raycast/api/index.js:8:2094)\n    at Suspense\n    at Nr (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/@raycast/api/index.js:7:2335)\n    at ray-root\n    at Oi (/Applications/Raycast.app/Contents/Resources/RaycastNodeExtensions_RaycastNodeExtensions.bundle/Contents/Resources/api/node_modules/@raycast/api/index.js:8:3103)","breadcrumbs":[]}: OAuth request creation is not available when command is launched in background
```

### Steps To Reproduce

1. Install Extension
2. Do any action (that requires authentication)
3. Select Apple SSO in linked web page
4. Sign in with your Apple-ID

### Current Behaviour

After Apple sign-in, you get redirected back to todoists sign-in page with an error "Invalid CSRF token"

### Expected Behaviour

You're logged in.

**Comment by thomaslombart** [comment]
Could you share a recording? It'll make it easier to see what's happening.

**Comment by max-zinn** [comment]
I swear the bug was there this morning! 🫨 

I just tried the same steps and suddenly I didn't even need to login. After click on "Sign in with Todoist" in raycast, the browser opened, todoist said I'm logged in and I got the Access-Grant screen. Since accepting that, it works.

It's still the same issue in the desktop todoist app though. But that's not your issue so I'm closing this.

Thanks for your help anyways.
